public/pages/CreateSnapshotPolicy/containers/CreateSnapshotPolicy/CreateSnapshotPolicy.tsx

```diff
@@ -0,0 +1,746 @@
+            }}
+            createRepo={this.createRepo}
+            snapshotManagementService={this.props.snapshotManagementService}
+            repoError={repoError}
+          />
+        </ContentPanel>
+
+        <EuiSpacer />
+
+        <ContentPanel title="Snapshot schedule" titleSize="m">
+          <CronSchedule
+            frequencyType={creationScheduleFrequencyType}
+            onChangeFrequencyType={(e) => {
+              this.setState({ creationScheduleFrequencyType: e.target.value });
+            }}
+            timezone={_.get(policy, "creation.schedule.cron.timezone", "America/Los_Angeles")}
+            onChangeTimezone={(e) => {
+              this.setState({ policy: this.setPolicyHelper("creation.schedule.cron.timezone", e.target.value) });
+            }}
+            cronExpression={_.get(policy, "creation.schedule.cron.expression", "")}
+            onCronExpressionChange={(expression: string) => {
+              this.setState({ policy: this.setPolicyHelper("creation.schedule.cron.expression", expression) });
+            }}
+          />
+        </ContentPanel>
+
+        <EuiSpacer />
+
+        <ContentPanel title="Retention period" titleSize="m">
+          <EuiRadioGroup
+            options={rententionEnableRadios}
+            idSelected={deleteConditionEnabled ? "retention_enabled" : "retention_disabled"}
+            onChange={(id) => {
+              this.setState({ deleteConditionEnabled: id === "retention_enabled" });
+            }}
+          />
+
+          <EuiSpacer size="m" />
+
+          {deleteConditionEnabled ? (
+            <>
+              <CustomLabel title="Maximum age of snapshots retained" />
+              <EuiFlexGroup>
+                <EuiFlexItem grow={false}>
+                  <EuiFieldNumber
+                    value={maxAgeNum}
+                    min={1}
+                    onChange={(e) => {
+                      this.setState({ maxAgeNum: parseInt(e.target.value) });
+                    }}
+                  />
+                </EuiFlexItem>
+                <EuiFlexItem style={{ maxWidth: 100 }}>
+                  <EuiSelect
+                    options={MAX_AGE_UNIT_OPTIONS}
+                    value={maxAgeUnit}
+                    onChange={(e) => {
+                      this.setState({ maxAgeUnit: e.target.value });
+                    }}
+                  />
+                </EuiFlexItem>
+              </EuiFlexGroup>
+              <EuiSpacer size="m" />
+              <EuiAccordion id="additional_delete_conditions" buttonContent="Additional settings">
+                <EuiSpacer size="m" />
+
+                <EuiText>Number of snapshots retained</EuiText>
+                <EuiSpacer size="s" />
+
+                <EuiFlexGroup alignItems="flexStart">
+                  <EuiFlexItem grow={false}>
+                    <CustomLabel title="Min count" />
+                    <EuiFormRow isInvalid={!!minCountError} error={minCountError}>
```

Comment:
The validation here doesn't seem to be working as expected. I can enter a value greater than the `Max count` in the `min` field without receiving a warning. Submitting the policy will result in an error toast though.
```
Failed to create snapshot policy: {"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"max_count should be bigger than min_count."}],"type":"illegal_argument_exception","reason":"max_count should be bigger than min_count."},"status":400}
```
This warning text is displayed when the `min` field is left blank though. `Min count should be bigger than 0.`

Reply:
Will let the backend deal with max should > min